test/e2e/lib/resources/kube.go

```diff
@@ -49,6 +49,10 @@ func SenderJob(name string, envVars []corev1.EnvVar) *batchv1.Job {
 	return baseJob(name, "sender", envVars)
 }
 
+func BrokerSenderJob(name string, envVars []corev1.EnvVar) *batchv1.Job {
+	return baseJob(name, "sender_gcpbroker", envVars)
```

Comment:
Can we rename it to make it less confusing? Instead of "sender-gcpbroker", maybe name it to "retryablesender"